File: Gas Station/Order-Schedule-Allocation Services/stock-service/src/main/java/net/javaguides/stockservice/dto/UpdateEvent.java
package net.javaguides.stockservice.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.Date;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class UpdateEvent {

    private String _id;
    private String orderId;
    private String fuelType;
    private int qty;

    private String status;

    private Date date;

    private int __v ;

    public String get_id() {
        return _id;
    }

    public void set_id(String _id) {
        this._id = _id;
    }

    public int get__v() {
        return __v;
    }

    public void set__v(int __v) {
        this.__v = __v;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public void setOrderId(String orderId) {
        this.orderId = orderId;
    }

    public void setFuelType(String fuelType) {
        this.fuelType = fuelType;
    }

    public void setQty(int qty) {
        this.qty = qty;
    }



    public Date getDate() {
        return date;
    }



    public void setDate(Date date) {
        this.date = date;
    }

    public String getOrderId() {
        return orderId;
    }

    public String getFuelType() {
        return fuelType;
    }

    public int getQty() {
        return qty;
    }


}

File: Gas Station/Dispatch Service src/src/model/updateEvent.ts
export class UpdateEvent {
    orderId :string;
    fuelType: string;
    qty:number;
    status:string;
    date:Date;

    UpdateEvent(){

    }

    public setorderId(orderId:string){
        this.orderId=orderId;
    }
    public setfuelType(fuelType:string){
        this.fuelType=fuelType;
    }
    public setQty(qty:number){
        this.qty=qty;
    }
    public setStatus(orderId:string){
        this.orderId=orderId;
    }
    public setDate(date:Date){
        this.date=date;
    }
    
   }
    